source/summary.md

```diff
@@ -0,0 +1,373 @@
+
+### Use better building materials
+
+The next most important thing, after locking in text-related accessibility, is to use the correct building materials.
+
+As already hinted, `<canvas>` can have some limitations for accessibility. It's great for high-performance rendering, but that performance is directly related to a lack of markup + DOM overhead that comes with more accessible materials, like HTML (or even SVG).
+
+As for virtually everything that is interactive, from annotations, tooltip-triggering elements, to the buttons in the plot tools, natively semantic HTML elements will be far better for accessibility than the elements that they currently are. As an example: a `<div>` isn't natively semantic at all, but a `<button>` is. Adding on-click event listeners to a div would make it listen to mouse events, but keyboard-driven assistive technologies can't reach those elements natively, like they do with a button. The general philosophy here in accessibility communities online is that if a native element can do what you're trying to do, you should use that element.
+
+[Sarah Higley writes](https://sarahmhigley.com/writing/tooltips-in-wcag-21/) about how "a major pitfall of approaching web design as a visual medium is conflating visual patterns with functional or interaction patterns." Her point is that thinking in terms of a visual language can sometimes bias web developers and designers into trying to solve all interactive problems using visual means. HTML has a [massive array](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantic_elements) of elements with functional semantics and they should be used whenever possible. Not only will assistive technologies have a better time, but so will automatic agents when parsing a website (like crawlers for search engines and foundational models).
+
+Plus, it is easier for someone else to interpret code that uses semantic markup. This will certainly have a downstream effect on future community contributions and new projects.
+
+### Build a flexible system
+
+People personalize their technology. And people also use software with a variety of different devices in different contexts. A flexible, compromising system is necessary for that software to be usable for all people in all places.
+
+There are some existing, relatively ubiquitous UI customizations that people commonly do that Bokeh needs to "watch" for. People commonly use "high contrast" modes, increase font size/weight/spacing, turn off motion/animations, turn on redundant encodings, and zoom/magnify. Media queries and other strategies in CSS (and even JavaScript) can be used to look out for these user-determinations. But also, an API should be implemented that allow someone to programmatically override the design of a visualization on their own (in cases when it might be hard to automatically determine what the user's needs).
```

Comment:
What is your take on having something like "accessibility action buttons or something" for example to change to a high-contrast theme, turn off animations, increase font size readily available while of course, allowing users to customise/override as needed?

Reply:
Ah seems the paragraph below addresses some of this